pandas/tests/series/test_operators.py

```diff
@@ -962,6 +962,36 @@ def test_timedelta64_ops_nat(self):
 
 
 class TestDatetimeSeriesArithmetic(object):
+    def test_sub_datetime64_not_ns(self):
+        # GH#7996
+        ser = Series(date_range('20130101', periods=3))
+        dt64 = np.datetime64('2013-01-01')
+        assert dt64.dtype == 'datetime64[D]'
+        res = ser - dt64
+        expected = pd.Series([Timedelta(days=0), Timedelta(days=1),
+                              Timedelta(days=2)])
+        tm.assert_series_equal(res, expected)
+
+        res = dt64 - ser
+        tm.assert_series_equal(res, -expected)
```

Comment:
can you parametrize

Reply:
Not without introducing a whole lot of extra boilerplate.

Reply:
not sure that is the case. you are repeating Series/DTI testing.

Reply:
OK, just changed.  It is much less obvious to a casual reader exactly what this is testing.